GVFS/GVFS.Service/GVFS.Service.Mac.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\GVFS.Build\GVFS.cs.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>GVFS.Service</RootNamespace>
+    <AssemblyName>GVFS.Service</AssemblyName>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <Platforms>x64</Platforms>
+    <RunTimeIdentifiers>osx-x64</RunTimeIdentifiers>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>

```

Comment:
`<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` is set in `..\GVFS.Build\GVFS.cs.props`:

https://github.com/Microsoft/VFSForGit/blob/d02c62832d512ab3ef91aeed5d0457b4675934da/GVFS/GVFS.Build/GVFS.cs.props#L15

And so I think we should be able to remove it here, @nickgra would you agree?

Reply:
removed it.